ktor-client/ktor-client-features/ktor-client-logging/common/src/io/ktor/client/features/logging/Logging.kt

```diff
@@ -24,12 +24,13 @@ import kotlinx.coroutines.*
 class Logging(
     val logger: Logger,
     var level: LogLevel,
-    var filters: List<(HttpRequestBuilder) -> Boolean>
+    val filters: List<(HttpClientCall) -> Boolean>,
```

Comment:
I'm not sure if it's ok to replace the request builder with the call here. The call will be created only after the network request, but we should print the request log before any network activity. 
Could you tell me if you have an idea of fixing this?